docs/tutorials/word2vec.ipynb

```diff
@@ -37,20 +37,20 @@
         "id": "AOpGoE2T-YXS"
```

Comment:
<p>Can you remove the extra <code>&gt;</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/text/pull/1229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Weird, the "ReviewNB" app didn't seem to link it directly to the line....

Line 109.  But looking at the source, it could just be because of the html replacement.  Maybe we should just revert all those changes.

Reply:
<p>Agreed, I'm not sure where that html encoding noise came from. Maybe because I did the edits in colab? I'll revert all the ones I find.</p>